### PR TITLE
Add FAQ: #dcc_ri_429

### PR DESCRIPTION
**Pull request by larswmh**
image
image

Internal Tracking ID: [EXPOSUREAPP-12622](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12622)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1626,6 +1626,16 @@
...
                                 ]
                             },
+                            {
+                                "title": "Error 'DCC_RI_429'",
+                                "anchor": "dcc_ri_429",
+                                "active": false,
+                                "textblock": [
+                                    "The error message with the code \"DCC_RI_429\" occurs if you try to reissue a certificate more than three times. This could happen, for example, if you want to have the certificate reissued on multiple devices. In this case, you can scan the QR code of the certificate from another device if you have already received the new, replaced certificate there.",
+                                    "Because of this limitation, we recommend that you export the newly issued certificate as a PDF file and print it out so that you can scan the QR code of the certificate again if needed",
```
Comment (Ein-Tim) [inline src/data/faq.json:1635]: I suggest to add a link to the FAQ entry explaining how to create certificate printouts.

File: src/data/faq_de.json [hunk 2]
```diff
@@ -1627,6 +1627,16 @@
...
                                 ]
                             },
+                            {
+                                "title": "Fehlermeldung 'DCC_RI_429'",
+                                "anchor": "dcc_ri_429",
+                                "active": false,
+                                "textblock": [
+                                    "Die Fehlermeldung mit dem Kode „DCC_RI_429“ tritt auf, wenn Sie mehr als drei Mal versuchen, ein Zertifikat neu auszustellen. Das könnte z.B. geschehen, wenn Sie sich auf mehreren Geräten das Zertifikat neu ausstellen lassen möchten. In diesem Fall können Sie den QR-Code des Zertifikates von einem anderen Gerät scannen, wenn Sie dort schon das neue, ausgetauschte Zertifikat erhalten haben.",
+                                    "Wir empfehlen Ihnen wegen dieser Beschränkung das neu ausgestellte Zertifikat als PDF-Datei zu exportieren und auszudrucken, damit Sie bei Bedarf den QR-Code des Zertifikats erneut einscannen können",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:1636]: See comment on English code.

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Please see the in-line comment.

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:1635]: excellent idea!

Comment (dsarkar) [reply src/data/faq.json]: link added.

**Comment by dsarkar** [comment]
also added missing dot.

**Review by Ein-Tim — APPROVED** [review]
LGTM! Did not review for typos, but only the content.

File: src/data/faq_de.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Thanks, same back to you @dsarkar!

**Comment by dsarkar** [comment]
EN
image

DE
image

**Review by dsarkar — APPROVED** [review]

**Review by GisoSchroederSAP — APPROVED** [review]
Looks good to me; easy wording, clear recommendations.
Approved.

**Comment by MikeMcC399** [comment]
@dsarkar 
- This PR writes `z.B.` without a space, which is not in line with DIN [DIN 5008](https://de.wikipedia.org/wiki/DIN_5008#Abk%C3%BCrzungen_(Abschnitt_4.5)). (See https://github.com/corona-warn-app/cwa-website/issues/1689)
- It does not seem worth submitting a PR just for this correction. Maybe combine with another planned PR?

**Comment by dsarkar** [comment]
@MikeMcC399 Sure, we will fix that with another PR. Thanks.

**Comment by dsarkar** [comment]
@MikeMcC399 white space fixed with https://github.com/corona-warn-app/cwa-website/pull/2724

**Pull request merged**